src/components/ApplicationGroup/Details/TriggerView/EnvTriggerView.tsx

```diff
@@ -216,6 +216,9 @@ export default function EnvTriggerView({ filteredAppIds, isVirtualEnv }: AppGrou
             setDefaultConfig(_isDefaultConfig)
             setConfigPresent(isConfigPresent)
         })
+        .catch((error) => {
```

Comment:
getDefaultConfig() makes a get call internally that throws a ServerError in case of errors. Thus there should be error handling.